tutorials/colors/colors.py

```diff
@@ -5,59 +5,105 @@
++--------------------------------------+--------------------------------------+
+| Case-insensitive color name from     | - ``'xkcd:sky blue'``                |
+| `xkcd color survey`_ with ``'xkcd:'``| - ``'xkcd:eggshell'``                |
+| prefix.                              |                                      |
++--------------------------------------+--------------------------------------+
+| Case-insensitive Tableau Colors from | - 'tab:blue'                         |
+| 'T10' categorical palette.           | - 'tab:orange'                       |
+|                                      | - 'tab:green'                        |
+|                                      | - 'tab:red'                          |
+|                                      | - 'tab:purple'                       |
+| .. note:: This is the default color  | - 'tab:brown'                        |
+|           cycle.                     | - 'tab:pink'                         |
+|                                      | - 'tab:gray'                         |
+|                                      | - 'tab:olive'                        |
+|                                      | - 'tab:cyan'                         |
++--------------------------------------+--------------------------------------+
+| "CN" color spec where ``'C'``        | - ``'C0'``                           |
+| precedes a number acting as an index | - ``'C1'``                           |
+| into the default property cycle.     +--------------------------------------+
+|                                      | :rc:`axes.prop_cycle`                |
+| .. note:: Indexing occurs at         |                                      |
+|           rendering time and defaults|                                      |
+|           to black if cycle does not |                                      |
+|           include color.             |                                      |
++--------------------------------------+--------------------------------------+
 
 .. _xkcd color survey: https://xkcd.com/color/rgb/
 
-"Red", "Green", and "Blue" are the intensities of those colors, the combination
-of which span the colorspace.
+"Red", "Green", and "Blue" are the intensities of those colors. In combination,
+they represent the colorspace.
 
-How "Alpha" behaves depends on the ``zorder`` of the Artist.  Higher
-``zorder`` Artists are drawn on top of lower Artists, and "Alpha" determines
-whether the lower artist is covered by the higher.
-If the old RGB of a pixel is ``RGBold`` and the RGB of the
-pixel of the Artist being added is ``RGBnew`` with Alpha ``alpha``,
-then the RGB of the pixel is updated to:
-``RGB = RGBOld * (1 - Alpha) + RGBnew * Alpha``.  Alpha
-of 1 means the old color is completely covered by the new Artist, Alpha of 0
-means that pixel of the Artist is transparent.
+Matplotlib composes Artists based on the ``zorder`` keyword argument. If there
```

Comment:
zorder only specifies the draw order (which one is on top). "Composing" is a bit too big a word here and a bit vague at the same time.

Also, I would use "parameter" instead of "keyword argument", because a keyword argument is only one way of setting the zorder; `set_zorder()` would be another way.